tree/20_cfe_basics/package_lib.cf

```diff
@@ -75,7 +75,7 @@ bundle agent ncf_package(name, version, architecture, provider, state, options)
 
       # Select the right package manager
       # The default one has already been selected if needed using mackage_module_knowledge
-      "use_apt_provider" expression => strcmp("apt", "${provider}");
+      "use_apt_provider" expression => strcmp("apt_get", "${provider}");
```

Comment:
When specifying the provider in the generic method the parameter is `apt`, so we should detect both as valid `apt` provider.

Reply:
you are totally right ! thank you